### PR TITLE
Fix: Makes the no data tabs more consistent

### PR DESCRIPTION
**Pull request by swaff**
WATER-2245

When displaying licence data, only show returns/charges/communications
tabs if there is data to show.

### CODE REVIEW

File: src/internal/views/nunjucks/view-licences/expired-licence.njk [hunk 1]
```diff
@@ -22,43 +22,48 @@
...
 
-      {% set returnsHtml %}
-        {% include "nunjucks/view-licences/tabs/returns.njk" %}
-      {% endset %}
+      {% if (returns.length > 0 or communications.length > 0 or (showChargeVersions and chargeVersions.length > 0)) %}
+        <div class="govuk-tabs" data-module="tabs">
+          <h2 class="govuk-tabs__title">Contents</h2>
+          <ul class="govuk-tabs__list">
+            {% if returns.length > 0 %}
+              <li class="govuk-tabs__list-item">
```
Comment (james-siteclick) [inline src/internal/views/nunjucks/view-licences/expired-licence.njk]: Does the govuk-tabs component automatically select the first link in the tab navigation?
The HTML for the component shows `govuk-tabs__list-item--selected` as the class on the first element

File: src/internal/views/nunjucks/view-licences/licence.njk [hunk 2]
```diff
@@ -28,15 +29,17 @@
             </a>
           </li>
 
-          {% if returns.length %}
+          {% if returns.length > 0 %}
             <li class="govuk-tabs__list-item">
```
Comment (james-siteclick) [inline src/internal/views/nunjucks/view-licences/licence.njk]: Same here, should the first LI have `govuk-tabs__list-item--selected` or is it preselected without it?

File: src/internal/views/nunjucks/view-licences/expired-licence.njk (hunk 1 above)
Comment (swaff) [reply src/internal/views/nunjucks/view-licences/expired-licence.njk]: The govuk component does set the selected tab.

File: src/internal/views/nunjucks/view-licences/licence.njk (hunk 2 above)
Comment (swaff) [reply src/internal/views/nunjucks/view-licences/licence.njk]: In this case the summary tab is always the selected tab.

**Pull request merged**